docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,352 @@ Review important information about the {kib} 8.x releases.
+* Settings add config to enable default rules ({kibana-pull}190800[#190800]).
+* Added alerts page ({kibana-pull}190751[#190751]).
+* Monitor list add bulk delete ({kibana-pull}190674[#190674]).
+* Delete monitor API via id param !! ({kibana-pull}190210[#190210]).
+* Enable metrics and traces in the Data Set Quality page ({kibana-pull}190043[#190043]).
+* Adds alert grouping functionality to the observability alerts page ({kibana-pull}189958[#189958]).
+* Adds a new SLO Burn Rate embeddable ({kibana-pull}189429[#189429]).
+* The Slack Web API Alert Connector is now supported as a default connector for Synthetics and Uptime rules ({kibana-pull}188437[#188437]).
+* Adds option to enable backfill transform ({kibana-pull}188379[#188379]).
+* Save the ECS group by fields at the AAD root level ({kibana-pull}188241[#188241]).
+* Adds last value aggregation ({kibana-pull}187082[#187082]).
+* Improve synthetics alerting ({kibana-pull}186585[#186585]).
+* Make overview grid embeddable ({kibana-pull}160597[#160597]).
+Elastic Security solution::
+For the Elastic Security 8.16.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+Kibana security::
+* Adds an API endpoint `POST security/roles` that can be used to bulk create or update roles ({kibana-pull}189173[#189173]).
+* Automatic Import can now create integrations for logs in the CSV format ({kibana-pull}194386[#194386]).
+* Adds an error handling framework to Automatic Import that provides error messages with more context to user ({kibana-pull}193577[#193577]).
+* When running in FIPS mode, Kibana forbids usage of PKCS12 configuration options ({kibana-pull}192627[#192627]).
+Machine Learning::
+* Adds new section for creating daylight saving time calendar events ({kibana-pull}193605[#193605]).
+* Anomaly Detection: Adds a page to list supplied job configurations ({kibana-pull}191564[#191564]).
+* Redesigns start/update model deployment dialog to support adaptive resources ({kibana-pull}190243[#190243]).
+* File upload: Adds support for PDF files ({kibana-pull}186956[#186956]).
+* Adds Pattern analysis embeddable for dashboards ({kibana-pull}186539[#186539]).
+Management::
+* This release introduces a fresh, modern look for the console, now featuring the Monaco editor. We've added a file import and export functionality, and the console is fully responsive with stackable panels for a smoother experience. New buttons allow for quick clearing of editor values and output. Additionally, the history and config tabs were improved to enhance usability. ({kibana-pull}189748[#189748]).
+
+For more information about the features introduced in 8.16.0, refer to <<whats-new,What's new in 8.16>>.
+
+[[enhancements-and-bug-fixes-v8.16.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.16.0 release, review the enhancements and bug fixes.
+
+
+[float]
+[[enhancement-v8.16.0]]
+=== Enhancements
+Alerting::
```

Comment:
For alerting, I think we should add the following enhancements. We forgot to add the release note due to multiple PRs working on each feature:

- New full-page rule form in the Stack Management page (https://github.com/elastic/kibana/pull/194655)
- Flapping settings per rule (https://github.com/elastic/kibana/pull/189341)

@lcawl Do you think the "Flapping settings per rule" should be part of the "Features" section?

Reply:
Done in b20d638e7ddf062aece7999254002bae5b12d984